scripts/gen-gh-pages-docs.py

```diff
@@ -109,11 +110,47 @@ def create_policy_docs(docs_file, output_dir):
     for scm in docs_yaml:
         create_scm_policy_docs(scm, docs_yaml[scm], output_dir)
 
-def print_usage():
-    print("python gen-gh-pages-docs.py docs_file output_directory")
+def create_monomarkdown(docs_file, output_dir):
+    table_of_contents = "# Table of contents\n"
 
-if __name__ == '__main__':
-    if len(sys.argv) != 3:
-        print_usage()
-        exit(1)
-    create_policy_docs(sys.argv[1], sys.argv[2])
+    docs_yaml = get_docs_yaml(docs_file)
+    result = ""
+
+    i = 1
+    for scm in docs_yaml:
+        pretty_name = scm_to_pretty_name(scm)
+        result += f"# {pretty_name}\n"
```

Comment:
i suggest creating a function called create md header so that the code could be more self explained. 
the function should get the header size. meaning (h1 - #, h2 - ##, ...) and the string